server/lib/utils/connstr-mssql.js

```diff
@@ -0,0 +1,40 @@
+import _ from "lodash";
+
+export function parseConnectionConfig(settings) {
```

Comment:
@sbellity, this would mean we write the same code twice, because PostgreSQL and MySQL use the same... DRY?! ;-)